### ISSUE TITLE
EmbeddedJedis doesn't support sinter command

### ISSUE THREAD

**Issue by charlesivie** [lordofthejars/nosql-unit#130]
trying to run EmbeddedJedis and using the sinter command gives me the following:

java.lang.UnsupportedOperationException: The method sinter with parameters [[Ljava.lang.String;@79179359] does not exist on class class com.lordofthejars.nosqlunit.redis.embedded.EmbeddedJedis

**Comment by lordofthejars** [comment]
Yes I know the problem with embeddedJedis is that guys changed almost all the interfaces so now it will require a deep change on the whole class and add new operations.

**Comment by charlesivie** [comment]
Thought it could be something like that. Is it possible to use a previous version of Jedis and get support for the sinter command with nosql-unit?

**Comment by lordofthejars** [comment]
Well the operation itself is not implemented and probably the latest
version of Jedis that was supported by embeddedJedis won't contain sinter
operation.

El dt., 31 març, 2015 a les 13:20, cicoders (<notifications@github.com>) va
escriure:
